dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -296,13 +312,24 @@ object Engine {
     model
   }
 
-  def init(
-    node: Int,
-    cores: Int,
-    onSpark: Boolean = false
-  ): Option[SparkConf] = {
```

Comment:
Instead of removing the old interface, we can mark it deprecated, print warnings, call the new method, and optionally check if the result consistent with the user input?